homeassistant/components/zwave_js/sensor.py

```diff
@@ -326,131 +328,162 @@
         native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
         device_class=SensorDeviceClass.SIGNAL_STRENGTH,
         state_class=SensorStateClass.MEASUREMENT,
+        convert=convert_dict_of_dicts,
     ),
-    SensorEntityDescription(
+    ZWaveJSStatisticsSensorEntityDescription(
         key="backgroundRSSI.channel2.average",
         name="Average background RSSI (channel 2)",
         native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
         device_class=SensorDeviceClass.SIGNAL_STRENGTH,
+        convert=convert_dict_of_dicts,
     ),
-    SensorEntityDescription(
+    ZWaveJSStatisticsSensorEntityDescription(
         key="backgroundRSSI.channel2.current",
         name="Current background RSSI (channel 2)",
         native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
         device_class=SensorDeviceClass.SIGNAL_STRENGTH,
         state_class=SensorStateClass.MEASUREMENT,
+        convert=convert_dict_of_dicts,
     ),
 ]
 
 # Node statistics descriptions
 ENTITY_DESCRIPTION_NODE_STATISTICS_LIST = [
-    SensorEntityDescription(
+    ZWaveJSStatisticsSensorEntityDescription(
         key="commandsRX",
         name="Successful commands (RX)",
         state_class=SensorStateClass.TOTAL,
     ),
-    SensorEntityDescription(
+    ZWaveJSStatisticsSensorEntityDescription(
         key="commandsTX",
         name="Successful commands (TX)",
         state_class=SensorStateClass.TOTAL,
     ),
-    SensorEntityDescription(
+    ZWaveJSStatisticsSensorEntityDescription(
         key="commandsDroppedRX",
         name="Commands dropped (RX)",
         state_class=SensorStateClass.TOTAL,
     ),
-    SensorEntityDescription(
+    ZWaveJSStatisticsSensorEntityDescription(
         key="commandsDroppedTX",
         name="Commands dropped (TX)",
         state_class=SensorStateClass.TOTAL,
     ),
-    SensorEntityDescription(
+    ZWaveJSStatisticsSensorEntityDescription(
         key="timeoutResponse",
         name="Timed out responses",
         state_class=SensorStateClass.TOTAL,
     ),
-    SensorEntityDescription(
+    ZWaveJSStatisticsSensorEntityDescription(
         key="rtt",
         name="Round Trip Time",
         native_unit_of_measurement=UnitOfTime.MILLISECONDS,
         device_class=SensorDeviceClass.DURATION,
         state_class=SensorStateClass.MEASUREMENT,
     ),
-    SensorEntityDescription(
+    ZWaveJSStatisticsSensorEntityDescription(
         key="rssi",
         name="RSSI",
         native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
         device_class=SensorDeviceClass.SIGNAL_STRENGTH,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    ZWaveJSStatisticsSensorEntityDescription(
+        key="lastSeen",
+        name="Last Seen",
+        device_class=SensorDeviceClass.TIMESTAMP,
+        convert=lambda statistics, value: datetime.fromisoformat(dt)  # type: ignore[arg-type]
+        if (dt := statistics.get(value))
```

Comment:
Please use parenthesis around the lambda and indent that so it's easier to distinguish it from the description attributes.